src/Delegates/HandlesIndex.php

```diff
@@ -30,12 +38,15 @@ public function deleteAllIndexes(): void
         }
     }
 
-    public function getIndex($uid): Indexes
+    public function getIndex(string $uid): Indexes
```

Comment:
Since it has nothing to do with PHPCS, can you apply these kind of changes about type in another PR please? 🙂

Reply:
@curquiza 

Main reason for doing it was because of the `declare(strict_types=1);` It's good to use types in PHP but even better to enforce them with that directive.

When I applied the strict_type unit tests started breaking https://github.com/meilisearch/meilisearch-php/pull/56/commits/7ceea76528879fbc0552eba990939c9b13a4678e (https://github.com/meilisearch/meilisearch-php/runs/834289995) and that's why i removed the signature from here.

So maybe i can do a PR with applying just the simple native_invocation & a different PR for introducing `declare(strict_types=1);` but keep in mind that currently if somebody passed a `int` to getDocument it will fail.

Reply:
Ok, I have'nt got that before, so I would rather have these changes in a new PR 🙂

Yes I keep in mind this is indeed a bug, I'm going to open an issue! Thanks @devrck for your explanation! 🚀

Reply:
Alright will do the changes requested :wink: and we tackle this `declare(strict_types=1);` in a different PR.